addons/website/static/src/js/menu/seo.js

```diff
@@ -724,20 +724,32 @@ var SeoConfigurator = Dialog.extend({
         } else {
             return {
                 model: m[1],
-                id: m[2]|0
+                id: m[2] | 0,
+            };
+        }
+    },
+    getSeoObject: function () {
+        var repr = $('html').data('seo-object');
```

Comment:
add unslug function unslug(str) { return (id, model) } and use it in getMainObject and getSeoObject instead to copy past all the code

Reply:
Well I actually used the `main_object_request` from the `website_root` where I also added the seo object request and encapsulated the code there for both method.

Now SEO.js will just trigger_up those event to get the value.